packages/block-library/src/avatar/index.php

```diff
@@ -87,11 +87,13 @@ function render_block_core_avatar( $attributes, $content, $block ) {
 /**
  * Generates class names and styles to apply the border support styles for
  * the Avatar block.
+ * //phpcs:disable Gutenberg.NamingConventions.ValidBlockLibraryFunctionName
```

Comment:
@gziolo To use the `WordPress.` in the rule's name, it would need to become a part of the [WordPress-Coding-Standards](https://github.com/WordPress/WordPress-Coding-Standards) package (`wpcs`). However, getting this rule into the `wpcs` package might be challenging.

`wpcs` is currently undergoing a major overhaul. It's possible that you may need to wait for an undefined period of time until it's released.

Since this rule is currently essential for Gutenberg, my suggestion is to commit it to Gutenberg first. If the `wpcs` contributors agree that it should be included in `wpcs`, it can always be removed from the Gutenberg package.

To address the concern about maintainability, I will relocate the `//phpcs:disable` and `//phpcs:enable` statements from the doc blocks to prevent any interference.

Reply:
We might have linting disabled on files copied from Gutenberg through WP packages, so you can double check that first before applying any changes. I guess we can live with the `Gutenberg.` specific rule if it doesn't throw errors on CI in WP core.

Reply:
@gziolo 
The `//phpcs:disable Gutenberg.NamingConventions.ValidBlockLibraryFunctionName` statements shouldn't generate any linter errors in Core. However, personally, I'm not fond of the idea of having these comments in Core code.
I think I will add the possibility to ignore specific functions via the linter rule's configuration in phpcs.xml.dist. This way, we can avoid cluttering Core code with these comments. I'll make the necessary updates to the PR and let you know.

Reply:
Maybe we could list these function names as exceptions in the config file. Not ideal, but it should work.

Reply:
I've updated the PR.
So it can be reviewed now. 
Thanks!